Makefile

```diff
@@ -26,14 +26,14 @@ fastly:
 
 # useful for attaching a debugger such as https://github.com/go-delve/delve
 debug:
-	@go build -gcflags="all=-N -l" -o "fastly" ./cmd/fastly
+	@go build -gcflags="all=-N -l" $(LDFLAGS) -o "fastly" ./cmd/fastly
```

Comment:
Added in the `LDFLAGS` throughout this file because otherwise when running `fastly version` with a local dev build of the CLI you would just get empty output.

Here's the difference between me running `make install` on the current `main` branch vs building on this PR's branch...

<img width="911" alt="Screenshot 2022-04-01 at 09 15 54" src="https://user-images.githubusercontent.com/180050/161223700-5c01fc99-177f-47fb-8f82-24ce719e2c9f.png">